amq/README.md

```diff
@@ -57,7 +57,6 @@ The CE-Testsuite is divided by profiles, to enable the **amq** profile all you n
   - -Dtest=**The test class name, if you want to run only one test, otherwise all tests will be executed**
     - -Dtest=testOpenWireConnection
   - -Dkubernetes.ignore.cleanup=true **(default is false), It will ignore the resources cleanup, so you can take a look in the used pods to troubleshooting**
-  - -DAbortOnFirstFailure=true **Abort the execution of the test upon the first failure.** This is useful for debugging/troubleshooting purposes. It will leave behind all created artifacts, like logs and the whole namespace/project. It's up to you do the cleaning.
```

Comment:
I added this feature a while back so that I could mainly debug the artifacts on the server when a test failed.

But... if there's a better way to do that (like debugging on Eclipse), that's ok to me.

My only comment then about this is that this removal could happen in a separated commit, explaining why it's being removed and the alternative solution to the problem this piece of code was fixing.